src/main/java/seedu/address/model/person/DisplayPhoto.java

```diff
@@ -63,7 +65,8 @@ public DisplayPhoto(String photoPath) throws IllegalValueException {
      */
     public static boolean isValidFile(String test) {
         File file = new File(test);
-        return file.exists() && file.length() <= MAX_SIZE;
+        return file.exists() && file.length() <= MAX_SIZE
+                && (test.endsWith(ALLOWED_TYPE_JPG) || test.endsWith(ALLOWED_TYPE_PNG));
```

Comment:
jpeg images end with both .jpg and .jpeg :(
you might be able to use the .jp*g regex, but im not 100% sure

Reply:
Updated ✔